src/components/modals/ListModal.tsx

```diff
@@ -72,7 +72,7 @@ export function ListModal<T>({
   return (
     <ModalUi4 title={title} bridge={bridge} onCancel={handleCancel}>
       {message == null ? null : <ModalMessage>{message}</ModalMessage>}
-      {textInput == null ? null : (
+      {textInput == null || !textInput ? null : (
```

Comment:
Since `textInput` has a default value up above, it will never be `null`. You can simply remove the `== null` check.